build.sbt

```diff
@@ -18,8 +18,8 @@ libraryDependencies ++= Seq(
   "io.circe"          %% "circe-generic"                    % "0.7.0",
   "com.beachape"      %% "enumeratum-circe"                 % "1.5.14",
   "com.amazonaws"     % "amazon-kinesis-client"             % "1.7.6",
-  "com.gu"            %% "content-api-client"               % "11.22",
-  "com.gu"            %% "editorial-production-metrics-lib" % "0.10"
+  "com.gu"            %% "content-api-client"               % "11.48",
+  "com.gu"            %% "editorial-production-metrics-lib" % "0.15-SNAPSHOT"
```

Comment:
I would advise against using a snapshot, we can just release a later version of the library.

Reply:
Yeah, this was just from when I was running it locally and waiting for the new version of the library to be released.